book/src/snapshot-verification.md

```diff
@@ -0,0 +1,34 @@
+# Verifying Snapshots and Account state proof
```

Comment:
Can you give more context? Perhaps a section titled "Problem" or "General Overview".  Describing the naive approach is sometimes helpful.  At a glance, looks like this might be a performance optimization over a simple hashing algorithm? In any case, not obvious without more context.